programs/drift/src/state/oracle.rs

```diff
@@ -154,7 +163,9 @@ pub fn get_pyth_price(price_oracle: &AccountInfo, clock_slot: u64) -> DriftResul
     let mut oracle_scale_div = 1;
 
     if oracle_precision > PRICE_PRECISION {
-        oracle_scale_div = oracle_precision.safe_div(PRICE_PRECISION)?;
+        oracle_scale_div = oracle_precision
+            .safe_div(PRICE_PRECISION)?
+            .safe_div(multiple)?;
```

Comment:
think you should directly change `oracle_precision` instead based on the multiple

that way it if/else comparison with `PRICE_PRECISION` is ok and oracle_conf also gets scaled appropriately